models/errors.js

```diff
@@ -38,11 +38,23 @@ module.exports = {
             super("Item");
         }
     },
+    ItemDoesNotHaveTagError: class ItemDoesNotHaveTagError extends NotFoundError {
+        constructor() {
+            super("That item does not have that tag");
```

Comment:
Would this result in the message being "The item does not have that tag not found"?  If so, could it be something else like "that tag on that item" so it ends up as "That tag on that item not found"?

Reply:
Oh whoops, fixing inheritance.